web/src/screens/private/Schedules/ViewSchedule.js

```diff
@@ -29,73 +23,114 @@ import {
   UPDATE_TIME_SEGMENT_MUTATION,
 } from '../../../graphql/time-segment.mutation';
 
-import TimeCountLabel from './TimeCountLabel';
-import ViewScheduleItem from './ViewScheduleItem';
-import TimeModal from '../../../components/Modals/TimeModal';
+import DayWeek from './components/DayWeek';
+import ScheduleWeekItem from './components/ScheduleWeekItem';
 import TableNextPrevious from '../../../components/Tables/TableNextPrevious';
+import ScheduleHeader from './components/ScheduleHeader';
+import SpreadPanel from '../../../components/Panels/SpreadPanel';
+import TimeRangeModal from '../../../components/Modals/TimeRangeModal';
+import ScheduleWeekHeader from './components/ScheduleWeekHeader';
 
-import styles from './ViewSchedule.styles';
+import styles from '../../../styles/AppStyle';
 
 class ViewSchedule extends React.Component {
   state = {
-    startTimeRange: moment().isoWeekday(1).startOf('week').unix(),
-    endTimeRange: moment().isoWeekday(1).startOf('week').add(7, 'days')
+    startTimeRange: moment()
+      .isoWeekday(1)
+      .startOf('week')
+      .add(1, 'days')
+      .unix(),
+    endTimeRange: moment()
+      .isoWeekday(1)
+      .startOf('week')
+      .add(8, 'days')
```

Comment:
add 8 days? might want to comment that to explain why. If i was bug hunting my brain picks this out as an anomaly

Reply:
First ISO day starts on Sunday - so startTimeRange is + 1 day. I'll move this to a selector.